bin/run-tests.sh

```diff
@@ -18,11 +18,14 @@ filenames="representation.txt mapping.json"
 for test_dir in testdata/*/*; do
     test_dir_name=$(basename "${test_dir}")
     test_dir_path=$(realpath "${test_dir}")
+    output_dir_path="/tmp/${test_dir_name}"
```

Comment:
This file was trying to write to the read only testdata folder. This had to be fixed to make the smoke tests succeed. Not sure how this worked before. 🤷‍♀️

Reply:
Odd!